tests/Unit/Elliptic/DiscontinuousGalerkin/Test_InitializeFirstOrderOperator.cpp

```diff
@@ -146,8 +147,55 @@ void check_compute_items(
     return ActionTesting::tag_is_retrievable<
         ElementArray<Dim, Metavariables<Dim>>, tag>(runner, element_id);
   };
+  // Volume items
   CHECK(tag_is_retrievable(
       ::Tags::Flux<ScalarFieldTag, tmpl::size_t<Dim>, Frame::Inertial>{}));
+  CHECK(tag_is_retrievable(::Tags::Flux<AuxiliaryFieldTag<Dim>,
```

Comment:
is this set of tags available in a typelist so that you can do a `tmpl::for_each`?

Reply:
changed to a typelist